openedx/adg/lms/webinar/admin.py

```diff
@@ -0,0 +1,17 @@
+"""
+Registering models for webinar app.
+"""
+from django.contrib import admin
+
+from .models import Webinar
+
+
+@admin.register(Webinar)
+class WebinarAdmin(admin.ModelAdmin):
+    """
+    Django admin class for Webinar
+    """
+
+    list_display = ('title', 'subject', 'start_time', 'presenter', 'status')
+    list_filter = ('start_time', 'presenter', 'status', )
+    search_fields = ('subject', 'title', 'language',)
```

Comment:
Instead of having a search field for language, wouldn't it be better to add it in `list_filter`?